lib/stat_tracker.rb

```diff
@@ -115,21 +115,58 @@ def create_team_goals_and_games
     #return the hash
   end
 
+  def calculate_game_stats
+    game_stats = {}
+
+    @game_teams.each do |game_team| 
+      game_id = game_team.game_id
+      team_id = game_team.team_id
+
+      game_stats[game_id] ||= { teams: {} }
+
+      game_stats[game_id][:teams][team_id] ||= { goals: 0, games: 0, shots: 0 }
+
+      game_stats[game_id][:teams][team_id][:goals] += game_team.goals
+      game_stats[game_id][:teams][team_id][:games] += 1
+      game_stats[game_id][:teams][team_id][:shots] += game_team.shots
+    end
+    game_stats
+  end
+
+  def identify_game_season 
```

Comment:
Is this to help check if the game is within the selected season? If so, Alora found another way to do that by looking if the first four digits of the game_id matched with the first four digits of the season